src/odemis/acq/stream/_sync.py

```diff
@@ -729,6 +730,8 @@ def _startLeeches(self, px_time, shape):
             except Exception:
                 logging.exception("Leech %s failed to start, will be disabled for this acquisition", l)
                 np = None
+                if self._dc_estimator is l:
```

Comment:
Maybe a comment would be useful what this line is checking for. Avoids that dc data is read at the end of acq?

Reply:
Done.